### PR TITLE
BZ-1764212 Additional information on ephemeral storage and direction …

### PR DESCRIPTION
**Pull request by lbarbeevargas**
…to existing documentation

### CODE REVIEW

**Comment by lbarbeevargas** [comment]
BZ-1764212 https://bugzilla.redhat.com/show_bug.cgi?id=1764212

@jsafrane PTAL for 3.11. Thank you!

**Comment by jsafrane** [comment]
lgtm

**Comment by lbarbeevargas** [comment]
@liangxia PTAL for 3.11. Thank you!

BZ-1764212 https://bugzilla.redhat.com/show_bug.cgi?id=1764212

File: architecture/additional_concepts/ephemeral-storage.adoc [hunk 1]
```diff
@@ -82,3 +82,26 @@ exists, the `root` partition does not hold any image layer or other writable sto
...
+[[section-managing-ephemeral-storage]]
+== Managing ephemeral storage
+
+Cluster administrators can manage ephemeral storage within a project by xref:../../admin_guide/quota.html#overview[setting quotas] that define the limit ranges and number of requests for ephemeral storage across all Pods in a non-terminal state. Developers can also xref:../../dev_guide/compute_resources.html#dev-compute-resources[set requests and limits] on this compute resource at the Pod and Container level.
+
+[[section-monitoring-ephemeral-storage]]
+== Monitoring ephemeral storage
+
+You can use `/bin/df` as a tool to monitor ephemeral storage usage on the volume where ephemeral container data is located, which is `/var/lib/docker` and `/var/lib/kubelet`. The available space for only `/var/lib/kubelet` is shown when you use the `df` command if `/var/lib/docker` is placed on a separate disk by the cluster administrator.
+
```
Comment (qinpingli) [inline architecture/additional_concepts/ephemeral-storage.adoc:95]: I checked the ephemeral storage on a 3.11 cluster, looks the dirs for ephemeral storage  are `/var/lib/docker` and `/var/lib/origin`.

Comment (lbarbeevargas) [reply architecture/additional_concepts/ephemeral-storage.adoc:95]: Thanks for checking this @qinpingli. I'll update the text from `/var/lib/kubelet` to `/var/lib/origin/`.

**Comment by lbarbeevargas** [comment]
@qinpingli Updated to reflect your feedback. PTAL. Thank you!

@jsafrane Per QE feedback, I updated `/var/lib/kubelet` to `/var/lib/origin`.

**Comment by qinpingli** [comment]
lgtm

**Comment by lbarbeevargas** [comment]
@openshift/team-documentation PTAL for 3.11 and add labels and milestone. 
https://bugzilla.redhat.com/show_bug.cgi?id=1764212

Waiting for Reporter to remove NEEDINFO tag on BZ before merging. I received his ack via Slack, but tag has not been removed yet. Thank you!

**Comment by ahardin-rh** [comment]
LGTM

**Comment by lbarbeevargas** [comment]
@ahardin-rh I removed the NEEDINFO request from the BZ. Please merge to enterprise-3.11. Thank you!

**Pull request merged**